packages/search/app/styles/navi-search.less

```diff
@@ -104,9 +104,9 @@ table.navi-search-result {
   cursor: not-allowed;
 }
 
-.navi-search-result > table > tbody > tr > td {
-  border-bottom: 1px solid rgba(@denali-grey-800, 0.2);
-  padding: 14px 10px;
+.navi-search-result td {
+  border-bottom: 1px solid rgba(@denali-grey-800, 0.2) !important;
+  padding: 14px 10px !important;
```

Comment:
Let's not use `!important`. It makes it much more difficult for future devs to deal with style conflicts. Let's instead apply another class to the `td` and make the selector more specific that way.